third_party/2and3/cryptography/hazmat/primitives/serialization/__init__.pyi

```diff
@@ -32,27 +32,27 @@ def load_ssh_public_key(
 ) -> Union[RSAPublicKey, DSAPublicKey, DHPublicKey, EllipticCurvePublicKey, Ed25519PublicKey]: ...
 
 class Encoding(Enum):
-    PEM: str
-    DER: str
-    OpenSSH: str
-    Raw: str
-    X962: str
+    PEM: Encoding
+    DER: Encoding
+    OpenSSH: Encoding
+    Raw: Encoding
+    X962: Encoding
 
```

Comment:
This doesn't look right to me. Enum values are usually annotated with the value type (`str` in this case), no the enum type. Otherwise the value type would be unknown, since it's not specified anywhere. I assume that type checkers special-case enums. Does using the value type cause any problems?

Reply:
Thank you for looking at PR. The reason I switched to the Enum type rather than `str` is because of functions like `third_party/2and3/cryptography/hazmat/primitives/asymmetric/rsa.pyi` have the following signature:

```python
class RSAPrivateKeyWithSerialization(RSAPrivateKey):
    @abstractmethod
    def private_bytes(
        self, encoding: Encoding, format: PrivateFormat, encryption_algorithm: KeySerializationEncryption
    ) -> bytes: ...
```

Rather than:

```
class RSAPrivateKeyWithSerialization(RSAPrivateKey):
    @abstractmethod
    def private_bytes(
        self, encoding: str, format: str, encryption_algorithm: KeySerializationEncryption
    ) -> bytes: ...
```

As in the first code block, `Encoding` and `PrivateFormat` refer to the Enums defined in `cryptography.hazmat.primitives.serialization`.

I am happy to separate this out into two different PRs if that would be better?

Reply:
This shouldn't be a problem. For example the following type checks fine with mypy, although `PlistFormat.FMT_XML` is marked as `int` in typeshed:

```python
from plistlib import PlistFormat

def foo(format: PlistFormat) -> None:
    pass

foo(PlistFormat.FMT_XML)
```

Reply:
Sure thing, I'll remove that alteration from the PR.